fuzz.py

```diff
@@ -46,6 +47,16 @@ def test_idempotent_any_syntatically_valid_python(
         # able to cope with it.  See issues #970, #1012, #1358, and #1557.
         # TODO: remove this try-except block when issues are resolved.
         return
+    except TokenError as e:
+        # The Python3.7 'compile' method accepts input that terminates with a
+        # multi-line continuation character (\), but black rejects it.
+        # TODO: remove this special-case once py37 reaches end-of-life
+        exception_message = e.args[0]
+        if (
+            src_contents.endswith("\\")
```

Comment:
Hrm.  Should probably also check that we are currently in a Python 3.7 environment here.